ql/src/test/org/apache/hadoop/hive/ql/exec/util/TestRetryable.java

```diff
@@ -340,7 +345,7 @@ public void testRetrySuccessSecureCallable() throws Throwable {
     ArgumentCaptor<PrivilegedExceptionAction> privilegedActionArgumentCaptor
       = ArgumentCaptor.forClass(PrivilegedExceptionAction.class);
     Mockito.verify(userGroupInformation,
-      Mockito.times(3)).doAs(privilegedActionArgumentCaptor.capture());
+      Mockito.times(1)).doAs(privilegedActionArgumentCaptor.capture());
```

Comment:
why did the value change?

Reply:
TBH that is a behaviour that I haven't found an answer yet: My best guess is that there is some strange behaviour with the argument captors and powermock. I remember that case from last November: when I just simple changed the test runner from PowerMockRunner to MockitoJUnitRunner, it started to provide 1 as an answer. I debugged the test and found out 1 is the correct answer. It really runs it only once. So that I reverted the change, debugged it and what I found? When I ran it with PowerMock again. the execution stopped only once. And still, the times(3) went green. 
I really don't know what PowerMock does in that case. And only at that test.